src/components/ContextMenu/component.tsx

```diff
@@ -1,16 +1,13 @@
 
 import React from 'react';
 import Element from './index';
-import ElementComponent, { ElementComponentProps } from '../Element/component';
+import BaseComponent from '../Element/component';
 
-interface ContextMenuProps extends ElementComponentProps {
-    items: Array<{ text: string, onClick: (e: any) => void }>,
-    dom?: any,
-    triggerElement?: any
-}
-
-class ContextMenu extends ElementComponent <ContextMenuProps, any> {
-    constructor(props: ContextMenuProps) {
+/**
+ * Represents a context menu. LEGACY: This is a legacy component and will be removed in the future. Use Menu instead.
```

Comment:
Ah interesting. I noticed this component before and wondered if the Code Editor should be using it (because it creates context menus). But if the intention is to delete it eventually, what is that gated on? Any reason not to remove it in the very near future?

Reply:
I can add an issue for this, it should be removed after it's usage has been removed from PCUI Graph.

Reply:
I can add an issue for this, it should be removed after it's usage has been removed from PCUI Graph.